packages/bpk-component-map/docs/migrating-from-v3-to-v4.md

```diff
@@ -0,0 +1,59 @@
+# Migrating from `v3` to `v4`
```

Comment:
This whole file 👏

Reply:
Thanks 😄 I based it off of one you wrote a while ago for another component.